src/help/zaphelp/contents/ui/tlmenu/edit.html

```diff
@@ -47,6 +30,12 @@ <H3>Previous</H3>
 and highlighted in the <a href="../tabs/request.html">Request</a> or
 <a href="../tabs/response.html">Response</a> tab as appropriate.
 
+<H3>Enable / Disable Forced User Mode</H3>
+This switches forced user mode on and off.<br/>
+The menu item is only enabled when you have defined a forced user for at least one 
+<a href="../start/concepts/contexts.html">context</a>, which can be done via the 
```

Comment:
Should go up one more parent (same for the following link).

Reply:
K